wdl/workflows/hiphase/hiphase.wdl

```diff
@@ -112,9 +112,12 @@ task run_hiphase {
 		RuntimeAttributes runtime_attributes
 	}
 
+	# to handle single samples with very high depth, we had to increase memory to 3*gpu
+	# to handle cohorts with very high depth, we had to increase memory to 4*gpu
 	Int threads = 16
-	Int mem_gb = threads * 2
+	Int mem_gb = threads * 4
```

Comment:
I mentioned this in slack, but if memory starts becoming an issue (by whatever definition), there are probably some knobs / levers we can tinker with to bring it down.